panels/user-accounts/um-photo-dialog.c

```diff
@@ -375,13 +375,39 @@ menu_item_for_filename (UmPhotoDialog *um,
         return menuitem;
 }
 
+static GSList *
+get_facesdirs ()
+{
+        GSList *facesdirs = NULL;
+        const gchar * const * data_dirs;
+        int i;
+        const char *override_facesdir;
+
+        override_facesdir = g_getenv ("CC_USER_ACCOUNTS_PANEL_FACESDIR");
+        if (override_facesdir == NULL || !g_strcmp0 (override_facesdir, "")) {
+                override_facesdir = USER_ACCOUNTS_PANEL_FACESDIR;
+        }
+
+        if (g_strcmp0 (override_facesdir, "")) {
+                facesdirs = g_slist_append (facesdirs, g_strdup (override_facesdir));
```

Comment:
It's more performant to use `g_slist_prepend()` and then use `g_slist_reverse()` when returning it.